src/app/shared/pipes/toArray.pipe.ts

```diff
@@ -0,0 +1,11 @@
+import { Pipe, PipeTransform } from '@angular/core';
```

Comment:
Didn't get what this pipe do

Could you clarify and define more clear name for pipe?